### PR TITLE
mark scope as provided

### PR DESCRIPTION
**Pull request by honnix**
@danielnorberg PTAL

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -181,6 +183,7 @@
         <groupId>io.norberg</groupId>
         <artifactId>auto-matter-jackson</artifactId>
         <version>0.14.1</version>
+        <scope>provided</scope>
```
Comment (danielnorberg) [inline pom.xml]: This must not be provided.

Comment (honnix) [reply pom.xml]: So ok.

**Comment by danielnorberg** [comment]
👍

**Comment by danielnorberg** [comment]
I guess they would. In which case maybe remove the redundant `provided` scope from the submodules?

**Comment by honnix** [comment]
Ah sure. I didn’t even check the sub modules. :D

**Comment by honnix** [comment]
i will close this PR because the children specify the scope already.

**Pull request closed without merging**